galsim/real.py

```diff
@@ -571,14 +620,26 @@ def _getFile(self, file_name):
             self.loaded_lock.release()
         return f
 
-    def getGal(self, i):
+    def getBandpass(self):
+        """Returns a Bandpass object for the catalog.
+        """
+        try:
+            bp = real_galaxy_bandpasses[self.band[0].upper()]
+        except KeyError:
+            raise ValueError("Bandpass not found.  To use bandpass '{0}', please add an entry to "
+                             "the galsim.real.real_galaxy_bandpasses "
+                             "dictionary.".format(self.band[0]))
```

Comment:
Probably worth adding a test of this.  i.e check that it raises a ValueError when some custom bandpass isn't in the dict.  Then add it in by hand in the test and check again, and it should work.